src/app/public/modules/toast/toaster.component.spec.ts

```diff
@@ -169,4 +170,17 @@ describe('Toast component', () => {
     toasts = getToastElements();
     expect(toasts.length).toEqual(0);
   }));
+
+  it('should prevent click events from bubbling beyond toast components', fakeAsync(() => {
```

Comment:
It might be worth just having a sanity check test here that ensure that if we watch the toast itself that clicks do get registered.

Reply:
done